src/components/Artboard.tsx

```diff
@@ -28,15 +32,18 @@ export default class Artboard extends React.Component<Props> {
   };
 
   render() {
+    const style = StyleSheet.flatten(this.props.style as UserStyles);
```

Comment:
do you need this type cast here? It's a bit dangerous

Reply:
I'm getting TypeScript errors without it (and in a number of other components too). I added it to get it to build.

Reply:
fixed it here: https://github.com/airbnb/react-sketchapp/pull/502